dds/DCPS/transport/framework/NetworkAddress.cpp

```diff
@@ -167,5 +185,45 @@ OPENDDS_STRING get_fully_qualified_hostname(ACE_INET_Addr* addr)
   return fullname;
 }
 
+bool set_socket_ttl(const ACE_SOCK_Dgram& unicast_socket, const char& ttl)
```

Comment:
The name 'unicast_socket' came from how it was used in RtpsUdpTransport/DataLink.  That name may just be confusing here.